python/semantic_kernel/utils/telemetry/model_diagnostics/gen_ai_attributes.py

```diff
@@ -1,32 +1,48 @@
 # Copyright (c) Microsoft. All rights reserved.
 
+from semantic_kernel.contents.utils.author_role import AuthorRole
+
 # Constants for tracing activities with semantic conventions.
 # Ideally, we should use the attributes from the semcov package.
 # However, many of the attributes are not yet available in the package,
 # so we define them here for now.
 
 # Activity tags
-SYSTEM = "gen_ai.system"
 OPERATION = "gen_ai.operation.name"
+SYSTEM = "gen_ai.system"
```

Comment:
Question: are there plans to have the `semantic-conventions` repo manage these `gen_ai_attributes` so we can depend on a package and not have to manage them ourselves?